rebar.config.script

```diff
@@ -0,0 +1,24 @@
+%% Reading the contents of the file
+case file:read_file("priv/swagger/swagger-initializer.js") of
```

Comment:
I'm hesitant about implementing this as a `rebar.config.script`. To me it should be a pre-compile hook.
I wonder if I run `rebar3 test` on this project… and it changes the `swagger-initializer.js`… wouldn't it be included in the subsequent commit and thus pushed to Github without the placeholder?
How do we avoid that?

Reply:
I was add to `gitginore` this file.. hope it will fix issue

Reply:
Not a good idea… we **do want** that file downloaded when people clones the repo.

Reply:
NO NO, this file is added as well and it is a part of repo. But the changes in this file should be ignored - something like that.

Reply:
What if we need to make changes to this file (like the automatic updater does)?
In that case, it will need to `git add -f priv/swagger/swagger-initializer.js`.
I don't think that's a good idea.
But then again, I don't know how to avoid changing the file in development mode (i.e. when working on this app) but do change it when this project is a dependency of another… 🤔 
A pre-compile hook will work for _making the change_, but it will need to be aware of the _environment_. I'm not sure if that's possible with `rebar3`. Maybe `rebar3` has an environment variable, like `REBAR3_PROJECT` and the script can check if it's `cowboy_swagger` or not… or it check if its current path includes `_build` (in which case, it can assume it's on another project)… 🤔

Reply:
Well... I have idea but not sure... if this will looks ok. So, we can use `swagger-initializer.template` in repo and ignore `swagger-initializer.js` - during run project it will be translated from `template` to `js` and just copy all data inputs from this file before make changes.

Reply:
Yeah… that's not a bad idea. What do you think, @paulo-ferraz-oliveira ?

Reply:
We'll need to adjust the github action to write `swagger-initializer.template` based on the `swagger-initializer.js` file that it downloads from the swagger repo.

Reply:
I'm Ok with the current implementation, but maybe if I have time, just before we decide to merge I'd like to test it out locally. I don't see documentation updated or have a really easy to understand example of how the new changes would be used.